test/integration/forms/form.js

```diff
@@ -1225,6 +1225,119 @@ context('Patient Action Form', function() {
       .should('contain', 'Test Field')
       .should('contain', 'Test field widget');
   });
+
+  specify('save and go back button', function() {
+    localStorage.setItem('form-state_11111', JSON.stringify({ saveButtonType: 'saveAndGoBack' }));
+
+    cy
+      .routeAction(fx => {
+        fx.data.id = '1';
+        fx.data.relationships.form.data = { id: '11111' };
+        fx.data.relationships['program-action'] = { data: { id: '11111' } };
+        fx.data.relationships['form-responses'].data = [
+          { id: '1', meta: { created_at: testTs() } },
+        ];
+
+        return fx;
+      })
+      .routeForm(_.identity, '11111')
+      .routeFormDefinition()
+      .routeFormActionFields()
+      .routeFormResponse(fx => {
+        fx.data.storyTime = 'Once upon a time...';
+
+        return fx;
+      })
+      .routeActionActivity()
+      .routePatientByAction()
+      .visit('/patient-action/1/form/11111')
+      .wait('@routeAction')
+      .wait('@routeForm')
+      .wait('@routePatientByAction')
+      .wait('@routeFormDefinition')
+      .wait('@routeFormResponse');
+
+    cy
+      .get('.form__controls')
+      .find('button')
+      .contains('Update')
+      .click()
+      .wait('@routeFormActionFields');
+
+    cy
+      .route({
+        status: 201,
+        method: 'POST',
+        delay: 100,
+        url: '/api/form-responses',
+        response: { data: { id: '12345' } },
+      })
+      .as('routePostResponse');
+
+    cy
+      .get('.form__controls')
+      .find('.js-save-button')
+      .should($el => expect($el.text().trim()).to.equal('Save + Go Back'));
```

Comment:
I was looking for a way to test the exact text value for the button element. `Save` vs. `Save + Go Back`. 

For instance, if the button has a text value of `'Save + Go Back'` and we test for `.contains('Save')`, that test could pass even when it shouldn't.

This method I used gets the jquery element's text and `trims()` the extra spaces from it. Maybe create a `.text()` command for this? Similar to [here](https://github.com/Lakitna/cypress-commands/blob/develop/docs/text.md).

Open to any better ways of doing this.

Reply:
Can we test for `Go Back` or not?

Reply:
Hmm... yes we could.

For the `Save + Go Back` button:

```js
.should('contain', 'Save + Go Back')
```

For the `Save` button:

```js
.should('contain', 'Save')
.should('not.contain', 'Go Back');
```

That makes sense to me.